server/tests/api/videos/video-transcoder.ts

```diff
@@ -281,6 +281,53 @@ describe('Test video transcoding', function () {
     }
   })
 
+  it('Should respect maximum bitrate values', async function () {
+    this.timeout(80000)
+
+    {
+      const videoAttributes = {
+        name: 'waiting video',
+        fixture: 'video_high_bitrate_1080p.mp4',
```

Comment:
You forgot to add the fixture in `server/tests/fixtures/video_high_bitrate_1080p.mp4`.

Reply:
Right, I keep forgetting to git add new files...

Reply:
The file is very large though. A few kB is the limit for fixtures, so that they don't clutter the repository. Hopefully we can make the video as short as we want.

Reply:
Any idea where I can get a good file for that? The current file is from http://www.jell.yfish.us/, but there is no license, so it's probably proprietary. And even if I cut the video to one second, it would be very large.